### PR TITLE
docs(angular) : improve angular example

### PR DESCRIPTION
**Pull request by santoshyadavdev**
The Angular example is not clear and does not work, this code adds an working example

### CODE REVIEW

File: todo-angular/angular-featurehub-app/src/app/feature-token.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { InjectionToken } from '@angular/core';
+
+export interface FeatureToken {
+  host: string;
+  apiKey: string;
+}
+
+export const FEATURE_FLAG_TOKEN = new InjectionToken<FeatureToken>('FEATURE_FALG_TOKEN');
```
Comment (IrinaSouth) [inline todo-angular/angular-featurehub-app/src/app/feature-token.ts]: could you please fix the typo 'FEATURE_FALG_TOKEN'  to  'FEATURE_FLAG_TOKEN'

File: todo-angular/angular-featurehub-app/src/app/app.module.ts [hunk 2]
```diff
@@ -1,8 +1,21 @@
-import { NgModule } from '@angular/core';
+import { APP_INITIALIZER, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { FeatureToken, FEATURE_FLAG_TOKEN } from './feature-token';
+import { FeatureHubFlagService } from './featuerhub-service';
```
Comment (IrinaSouth) [inline todo-angular/angular-featurehub-app/src/app/app.module.ts]: could you please fix typo './featuerhub-service'  to './featurehub-service' and rename that file accordingly. Thanks!

**Comment by IrinaSouth** [comment]
this is great @santoshyadavdev! thanks for doing this PR! Just minor comments around typos.

**Comment by santoshyadavdev** [comment]
> this is great @santoshyadavdev! thanks for doing this PR! Just minor comments around typos.

Done @IrinaSouth 😊

**Pull request merged**